app/models/job.rb

```diff
@@ -3,4 +3,7 @@ class Job < ActiveRecord::Base
   STATUSES = [:draft, :published, :archived]
   enumerize :state, in: STATUSES
   belongs_to :organization
+  belongs_to :published_by, foreign_key: :user_id, class_name: "User"
```

Comment:
Thanks for explaining, your argument makes total sense :+1: 

> As an alternative, we could define that `belongs_to` relationship with `publisher` (or even `author`) instead of `published_by`.

I'm leaning towards using `author` as any created item (be it a job, an event or a news_item) doesn't have to end up being published for whatever reason, if not then `publisher` would be my second pick :)